openfpga_flow/tasks/fpga_bitstream/overwrite_bitstream/device_4x4/config/test.openfpga

```diff
@@ -0,0 +1,53 @@
+# This file is copied from fix_device_example_script.openfpga
```

Comment:
All the openfpga shell script are now placed under openfpga_flow/openfpga_shell_scripts/

Reply:
This is special openfpga, which run special script within the flow. Do not plan to make the change